components/places/examples/sync_history.rs

```diff
@@ -2,29 +2,9 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-extern crate fxa_client;
-extern crate sync15_adapter;
-extern crate url;
-
-extern crate serde;
-#[macro_use]
-extern crate serde_derive;
-extern crate serde_json;
-
-extern crate rusqlite;
-
-extern crate clap;
-
-#[macro_use]
-extern crate log;
-extern crate env_logger;
-extern crate failure;
-
-extern crate places;
-
 use failure::Fail;
-
 use fxa_client::{AccessTokenInfo, Config, FirefoxAccount};
+use log::*;
```

Comment:
Worth noting we might want to just do `#[macro_use] extern crate log;` in the crate root (which still works and gives us the current semantics) to import these globally rather than add `use log::*` to most modules (worth noting that `use log::*` brings in more than just the macros).

We also could just start doing e.g. `log::warn!`, which IIUC works?

In the short term it doesn't really matter, it's your call what to do.

Reply:
I quite like `log::warn!` but don't feel strongly.

Reply:
Same here I like the `log::foo` form, will make the change tomorrow if no one feels strongly against it.